src/components/DocWrapper/DocList/DocList.test.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import DocList, {DocListProps} from './index';
+
+describe('DocList', () => {
+  let props: DocListProps;
+
+  beforeEach(() => {
+    props = {
+      className: 'test-class',
+      variant: 'ul',
+    };
+  });
```

Comment:
What's the benefit of this? 🤔 If you need the same props every time, why use `beforeEach`?

Reply:
I fixed it and just initialized the props variable with the values